README.md

```diff
@@ -15,12 +15,18 @@ For further details see `requirements.txt`.
 * For example:
```

Comment:
As in the `setup.py`, it would be nice to correct `README.md` as it also lists full requirements. You should just list the general ones.